ci-operator/templates/openshift/installer/cluster-launch-installer-src.yaml

```diff
@@ -0,0 +1,355 @@
+kind: Template
+apiVersion: template.openshift.io/v1
+
+parameters:
+- name: JOB_NAME_SAFE
+  required: true
+- name: JOB_NAME_HASH
+  required: true
+- name: NAMESPACE
+  required: true
+- name: IMAGE_FORMAT
+  required: true
+- name: IMAGE_INSTALLER
+  required: true
+- name: LOCAL_IMAGE_SRC 
+  required: true
```

Comment:
```diff
-- name: IMAGE_TESTS
+- name: LOCAL_IMAGE_SRC
+  required: true
+- name: IMAGE_CLI
```

`IMAGE_CLI` needs to be a parameter of the template or we won't get its value from ci-operator.